### PR TITLE
Add user disclosure.

### PR DESCRIPTION
**Pull request by MichaelRoytman**
### Description

This pull request adds a disclosure that appears before the Xpert chatbot is functional. This disclosure must be shown to the user before they are able to use the chatbot. This pull request also improves the overall styling of the chatbot UI.

NOTE: This release requires that the `PRIVACY_POLICY_URL` environment variable is added to the [frontend-app-learning](https://github.com/openedx/frontend-app-learning) configuration. This will be added when the release associated with this pull request is installed in to [frontend-app-learning](https://github.com/openedx/frontend-app-learning).

**JIRA**: [MST-2048](https://2u-internal.atlassian.net/browse/MST-2048) (private)

### Demo

https://github.com/edx/frontend-lib-learning-assistant/assets/11871801/9ddb6ac6-1bd6-4704-9b00-c1cff08adeb3

### CODE REVIEW

File: src/components/Disclosure/index.jsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            <li>How should an essay be structured?</li>
+            <li>How does photosynthesis work?</li>
+          </ul>
+        </div>
+      </div>
+      <p className="text-light-100 py-3">
+        <strong>Note: </strong>
+        This chat is AI generated (powered by ChatGPT). Mistakes are possible.
+        By using it you agree that edX may create a record of this chat.
+        Your personal data will be used as described in our&nbsp;
```
Comment (MichaelRoytman) [inline src/components/Disclosure/index.jsx:46]: The linter was complaining about using a trailing space here, so I used `&nbsp;`.

File: src/components/Sidebar/Sidebar.scss [hunk 2]
```diff
@@ -1,20 +1,19 @@
+@use '../../utils/variables';
+
 .sidebar {
-  height: 100vh;
-  /* Set the height of the chat container */
-  overflow-y: scroll;
-  /* Enable vertical scrolling */
-  scroll-behavior: smooth;
-  /* Add smooth scrolling behavior */
+  box-shadow: 0 0 1rem 0 #00000026,0 0 .625rem 0 #00000026;
```
Comment (MichaelRoytman) [inline src/components/Sidebar/Sidebar.scss:4]: The box shadow is sexier than a left border.

File: src/components/Sidebar/Sidebar.scss [hunk 3]
```diff
@@ -1,20 +1,19 @@
...
+  background-color: white;
 
-  &.open {
-    width: 30%;
-    right: 0;
-  }
+  width: 30%;
+  right: 0;
+
+  /* Enable vertical scrolling */
```
Comment (MichaelRoytman) [inline src/components/Sidebar/Sidebar.scss:12]: Whoops. I'll remove this post-review.

File: src/components/Sidebar/Sidebar.scss [hunk 4]
```diff
@@ -1,20 +1,19 @@
...
-  /* Add smooth scrolling behavior */
+  box-shadow: 0 0 1rem 0 #00000026,0 0 .625rem 0 #00000026;
   top: 0;
   bottom: 0;
-  border-left-width: 1px;
-  border-color: #9CA3AF;
-  background-color: #F9FAFB;
+  background-color: white;
 
-  &.open {
```
Comment (MichaelRoytman) [inline src/components/Sidebar/Sidebar.scss:14]: This open class had no effect.

File: src/components/Disclosure/index.jsx (hunk 1 above)
Comment (spencertiberi) [reply src/components/Disclosure/index.jsx:46]: I don't know if the linter likes this, but you could also use something like this in jsx:

```
{"This chat is AI generated (powered by ChatGPT). Mistakes are possible.
        By using it you agree that edX may create a record of this chat.
        Your personal data will be used as described in "}
```

File: src/components/Message/Message.scss [hunk 5]
```diff
@@ -1,16 +1,17 @@
...
 
   &.user {
-    background:#EBEBEB;
+    background: variables.$dark-green;
   }
 
   &.assistant {
-    background:#0A3055;
-    color: white;
+    background: #F2F0EF;
```
Comment (spencertiberi) [inline src/components/Message/Message.scss:14]: Any reason this isn't a variable too?

**Review by alangsto — APPROVED** [review]
LGTM 👍

**Review by spencertiberi — APPROVED** [review]

File: src/components/Message/Message.scss (hunk 5 above)
Comment (MichaelRoytman) [reply src/components/Message/Message.scss:14]: I didn't think we were using it anywhere else, so I didn't put it in the variables file. I could, though. Do you think we'd use the color elsewhere? There are a few sporadic colors here and there that I've just left out of the variables file.

File: src/components/Disclosure/index.jsx (hunk 1 above)
Comment (MichaelRoytman) [reply src/components/Disclosure/index.jsx:46]: Hm. That didn't work for me, unfortunately.

Comment (spencertiberi) [reply src/components/Disclosure/index.jsx:46]: How weird!

**Pull request merged**